test/octalpermissions-success.yml

```diff
@@ -17,3 +17,13 @@
       file:
         path: foo
         mode: 0123
+
+    - name: octal permissions test success (4000)
+      file:
+        path: baz
+        mode: 4000
```

Comment:
Gotcha, I'll fix the regexp and tests and update this PR. Thanks for explaining it to me!